imagemounter/volume_system.py

```diff
@@ -533,7 +533,7 @@ def detect(self, volume_system, vstype='detect'):
                 cur_v.info['label'] = line.replace("LV Name", "").strip()
             if "LV Size" in line:
                 size, unit = line.replace("LV Size", "").strip().split(" ", 1)
-                cur_v.size = int(float(size.replace(',', '.')) * {'KiB': 1024, 'MiB': 1024 ** 2,
+                cur_v.size = int(float(re.sub(r'[^0-9.]', "", size.replace(',', '.'))) * {'KiB': 1024, 'MiB': 1024 ** 2,
```

Comment:
Do you perhaps have an example line where this fails? It's OK if you haven't but it would be helpful for some tests in the future.

Reply:
In some cases, `lvm lvdisplay` may output other characters in the LV Size section.

Example:
`LV Size                <19.00 GiB`

In this case, `<` is included in the `size` variable and would throw an error when being converted to a float. The regular expression removes that character so no error occurs.